src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -582,9 +582,21 @@
     <Field Name="GetFieldFromHandle" Type="MethodSymbol?"/>
   </Node>
 
-  <Node Name="BoundDefaultExpression" Base="BoundExpression">
-    <!-- Type is null in the case of a default literal, and non-null for default(T). -->
+  <!--
+  Default literals can convert to the target type.
+  Does not survive initial lowering.
+  -->
+   <Node Name="BoundDefaultLiteral" Base="BoundExpression">
+    <!-- Type is null. -->
     <Field Name="Type" Type="TypeSymbol?" Override="true"/>
```

Comment:
>/> [](start = 57, length = 2)

Null="always" ?

Reply:
Thanks

---
In reply to: [315417046](https://github.com/dotnet/roslyn/pull/37596#discussion_r315417046) [](ancestors = 315417046)